.github/workflows/assign-new-prs.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Assign New PR
+on:
+  pull_request_target:
+    types: [opened]
+    branch:
+      - main
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  initial-review:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    name: Initial Review
+    steps:
+    - name: Assign the New Pull Request
+      uses: srggrs/assign-one-project-github-action@1.2.1

```

Comment:
[Version 1.2.1](https://github.com/srggrs/assign-one-project-github-action/releases/tag/1.2.1) was released on 2021-04-02, but the latest on the Marketplace is [version 1.3.1](https://github.com/srggrs/assign-one-project-github-action/releases/tag/1.3.1), released 2021-10-04. I believe we should use the latest version unless there's some reason it's damaged. Occurs below.

(I diffed these versions and saw nothing concerning.)

Usually I would validate and push changes for such a simple issue, but I have no way to validate this as my own fork doesn't have a Classic Project and I can't create one now.

Reply:
Done